examples/3rdparty/python/BUILD

```diff
@@ -4,6 +4,36 @@
 # see/edit requirements.txt in this directory to change deps.
 python_requirements()
 
+# NB: google-cloud-bigquery has the following prelude upon import:
+#
+# from pkg_resources import get_distribution
+# __version__ == get_distribution("google-cloud-bigquery").version
+#
+# which will fail at import time with pkg_resources.DistributionNotFound.
+# The argument `prepend_to_pythonpath=True` will work around this particular error by ensuring the
+# requirements are within a pex file located at the at the front of the PEX_PATH when executing
```

Comment:
at the front

Should this comment be incorporated into the help for this argument on the target?

Reply:
Yes, thank you! I've now done this for the `PythonRequirementLibrary.__init__()` docstring!